.github/workflows/upgrade.yaml

```diff
@@ -1,154 +0,0 @@
-name: Upgrade emqx operator
-
-concurrency:
-  group: upgrade-${{ github.event_name }}-${{ github.ref }}
-  cancel-in-progress: true
-
-on:
-  pull_request:
-
-jobs:
-  upgrade:
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        install:
-          - static
-          - helm
-        apiVersion:
-          - v1beta2
```

Comment:
Do not delete this file, should change v1beta2 to v1beta3 in this line

Reply:
done